### PR TITLE
Fixing package.json so 'npm run electron' will work 

### PR DESCRIPTION
**Pull request by martinbajalan**
"Fixing package.json so 'npm run electron' will work as per instructions in README file.”

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "build": "webpack --content-base client/ --progress --profile --colors --display-error-details --display-cached",
     "watch": "./node_modules/.bin/webpack-dev-server --content-base client/  --progress --colors --no-info --config webpack.config.js",
-    "start": "ENVIRONMENT=DEV electron electron/main"
+    "electron": "ENVIRONMENT=DEV electron electron/main"
```
Comment (andrewjaykeller) [inline package.json:19]: I think you should change the readme NOT this file.

You run with `npm start`

**Comment by tashoecraft** [comment]
The readme was out of date, not the code base. Closing this as readme was just updated for it.

**Comment by teonbrooks** [comment]
thanks for pointing this out @martinbajalan!

**Comment by martinbajalan** [comment]
thank you.

**Pull request closed without merging**